### ISSUE TITLE
[Accessibility] ToolStripStatusLabel BoundingRectangle property is null

### ISSUE THREAD

**Issue by Ashley-Li** [dotnet/winforms#3884]
* .NET Core Version:  .Net SDK 5.0.100-rc.2.20460.32

* Have you experienced this same bug with .NET Framework?
Yes

* More info:
This issue can reproduce on ToolStrip.

**Minimal repro:**
1. Extract “[testApp.zip](https://github.com/dotnet/winforms/files/5207198/testApp.zip)” and open it in VS.
2. Build and run the project.
3. Open Accessibility Insights For Windows tool, select focus on form and view result.

**Actual behavior**
Accessibility Insights For Windows tool complains about ToolStripStatusLabel BoundingRectangle property being null (508 502.3.1).
![image](https://user-images.githubusercontent.com/56664234/92879854-fd6b3500-f43f-11ea-8954-b87da53296d2.png)

**Expected behavior:**
ToolStripStatusLabel should have all required properties set.
